hazelcast/src/main/java/com/hazelcast/client/properties/ClientProperty.java

```diff
@@ -311,6 +311,12 @@ public final class ClientProperty {
     public static final HazelcastProperty METRICS_COLLECTION_FREQUENCY
             = new HazelcastProperty("hazelcast.client.metrics.collection.frequency", 5);
 
+    /**
+     * The default size of multipart job upload
+     */
+    public static final HazelcastProperty JOB_UPLOAD_PART_SIZE
+            = new HazelcastProperty("hazelcast.jobupload.partsize", 10_000_000);
```

Comment:
maxPartSize?

Reply:
This field defines the default part size if the value is not specified by the client. The client may choose any max part size as long as binary protocol can handle it